sourcebundle/builder.go

```diff
@@ -73,12 +73,16 @@ type Builder struct {
 	// selected version of each module registry package.
 	resolvedRegistry map[registryPackageVersion]sourceaddrs.RemoteSource
 
+	// resolvedRegistryVersionDeprecations tracks potential deprecations for
+	// each package version
```

Comment:
This comment is currently just restating the variable name, and some more detail would be helpful. Example future readers' questions you may want to proactively answer (or not! whatever you think is pertinent):

- Where is the data gathered from?
- What consumes this data?
- What's a "potential" deprecation?

While we're here, the field in the comment doesn't match the field name in the struct.

Reply:
Included 👍 https://github.com/hashicorp/go-slug/pull/59/commits/be5d90781af6a49d0b5f5dc5fca24e185629c14a